src/styles/page.scss

```diff
@@ -0,0 +1,13 @@
+* {
+  font-family: Roboto, sans-serif;
+  display: block;
```

Comment:
don't set it for * selector, use tags directly

Reply:
it's better not to apply any other styles to `*` selector except `box-sizing`